cpp/modmesh/transform/fourier.cpp

```diff
@@ -3,9 +3,6 @@
 namespace modmesh
 {
 
-namespace transform
```

Comment:
It is good to remove the namespace `transform`.  Under `modmesh`, there should not be many namespaces.